src/Events/Authorization/XacmlMapperHelper.cs

```diff
@@ -79,4 +81,41 @@ public static (string AttributeId, string AttributeValue) SplitResourceInTwoPart
 
         return (id, value);
     }
+
+    /// <summary>
+    /// Maps a subject in the provided CloudEvent to a XACML resource attribute in the provided resource category instance.
+    /// </summary>
+    /// <remarks>
+    /// A recognized subject property value begins with one of the prefixes "urn:altinn:organization:identifier-no",
+    /// or "urn:altinn:person:identifier-no", which is copied to the resource attribute with the corresponding URN.
+    /// This will enable the PDP to enrich the request with roles etc. that the user has in the context of the CloudEvent
+    /// subject (aka reportee).
+    ///
+    /// Also note that this requires a XACML subject attribute that the PDP understands in order to look up the user's
+    /// roles/access groups for that particular reportee, eg. "urn:altinn:userid" or "urn:altinn:person:identifier-no".
+    /// </remarks>
+    /// <param name="cloudEvent">CloudEvent containing a subject representing a party (ie. reportee)</param>
+    /// <param name="resourceCategory">The XACML resource category to be populated</param>
+    public static void AddResourceReporteeAttributeFromCloudEventSubject(CloudEvent cloudEvent, XacmlJsonCategory resourceCategory)
```

Comment:
Can we consider renaming the function? The term reportee is not much used in the platform components we are responsible for. 
```suggestion
    public static void AddResourceSubjectReporteeAttributeFromCloudEvent(CloudEvent cloudEvent, XacmlJsonCategory resourceCategory)
```

Reply:
Changed to `AddResourcePartyAttributeFromCloudEventSubject`